src/tests/JIT/HardwareIntrinsics/X86/Avx512F/ConvertToVector512Double.Int32.cs

```diff
@@ -15,14 +15,14 @@
 using System.Runtime.Intrinsics.X86;
 using Xunit;
 
-namespace JIT.HardwareIntrinsics.X86._Avx1.handwritten
+namespace JIT.HardwareIntrinsics.X86._Avx512F.handwritten
 {
     public static partial class Program
     {
         [Fact]
-        public static void ConvertToVector256SingleInt32()
+        public static void ConvertToVector512DoubleInt32()
         {
-            var test = new SimpleUnaryOpTest__ConvertToVector256SingleInt32();
```

Comment:
What happened to the 256 tests?

Reply:
This is detected as a file copy and modification when it is in fact a net new file.